react/src/quiz/quiz-components.tsx

```diff
@@ -71,11 +71,16 @@ export function DownloadUpload(): ReactNode {
     return (
         <div style={{ marginTop: '5px' }}>
             <button onClick={() => { exportQuizPersona() }}>
-                Download Quiz History
+                Export Quiz History
             </button>
             {' '}
             <button onClick={() => { void importQuizPersona() }}>
-                Upload Quiz History
+                Restore Quiz History
+            </button>
+            {/* Delete is mostly for testing. May want to comment out otherwise. */}
```

Comment:
We can go ahead and just remove all of the Delete functionality.

Mostly just a footgun for users, for testing it's pretty easy to open up the devtools and delete the localstorage entries.

Reply:
My main reason for keeping this was to allow merging preferentially in favor of local data. Maybe we could break out these options into a "manage data" page? Especially if urbanstats is going to have more stuff like this.

Reply:
I think a delete button is a bit misleading also because it doesn't do the whole euro thing of deleting you from the server. I think let's remove it and open an issue to create a manage data page in future

Reply:
This is a little hacky but maybe we could use another `prompt` to ask the user if they want to replace or merge their data?

Reply:
That could work. Though yeah a "manage data" page is the better solution. Maybe get this workable, merge, and then I can start drafting that?

Reply:
Sounds like a plan!